src/app/tasks/modals/grade-task-modal/grade-task-modal.component.html

```diff
@@ -0,0 +1,75 @@
+<mat-card class="grade-task-modal">
+  <mat-card-title>
+    Assess Task Quality
+  </mat-card-title>
+  <mat-card-content>
+    <!-- For Grade Tasks -->
+    <div *ngIf="task.definition.is_graded">
+      <p>
+        Please provide a grade for task
+        <label class="task-label">
+          {{task.definition.abbreviation}}
+        </label>
+      </p>
+      <div class="grade-toggle-group">
+        <mat-button-toggle-group
+          #group="matButtonToggleGroup"
+          (change)="updateGrade($event.value)"
+        >
+          <mat-button-toggle
+            *ngFor="let idx of gradeValues"
+            class="grade-toggle"
+            value="{{idx}}"
+            aria-label="{{grades[idx]}}"
+            matTooltip="This student completed the task to a {{grades[idx]}} standard"
+            [matTooltipShowDelay]="500"
+            aria-describedby="This student completed the task to a {{grades[idx]}} standard"
+          >
+            <grade-icon [grade]="grades[idx]"></grade-icon>
+          </mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+    </div>
+    <!-- For Quality Points Tasks -->
```

Comment:
**Suggestion**: Could we add margin between Grade Tasks and Quality points tasks if we have both. I think it should look better.

![image](https://user-images.githubusercontent.com/33137074/168466012-6daefb17-9527-431b-a51f-50c577441b72.png)

Reply:
Hi Ray, I was under the impression a task could only be graded or rated, not both. Since a task can be set with both options, I'm assuming it can be? I will confirm with Andrew. Thanks for pointing this out!

Reply:
> Hi Ray, I was under the impression a task could only be graded or rated, not both. Since a task can be set with both options, I'm assuming it can be? I will confirm with Andrew. Thanks for pointing this out!

Actually one of my unit used both quality and grade, so I tested it when review.

Reply:
Oh interesting, thanks for pointing it out, I'll fix this up.